### PR TITLE
test preview push of docs

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by fredrikekre** [comment]
@oxinabox, the addition in this PR pushes to `preview-PR#`, e.g. https://fredrikekre.github.io/Literate.jl/preview-PR75/ in this case.

File: docs/make.jl [hunk 1]
```diff
@@ -50,6 +50,25 @@ makedocs(
...
 )
 
+############################################
+# Set up for pushing preview docs from PRs #
+############################################
+if haskey(ENV, "TRAVIS_PULL_REQUEST") && ENV["TRAVIS_PULL_REQUEST"] != "false"
+    @info "Pushing preview docs."
+    PR = ENV["TRAVIS_PULL_REQUEST"]
+    # Overwrite Documenter's function for generating the versions.js file
+    foreach(Base.delete_method, methods(Documenter.Writers.HTMLWriter.generate_version_file))
```
Comment (oxinabox) [inline docs/make.jl]: Oh, that is wild.

File: docs/make.jl [hunk 2]
```diff
@@ -50,6 +50,25 @@ makedocs(
         "generated/example.md"]
 )
 
+############################################
+# Set up for pushing preview docs from PRs #
+############################################
+if haskey(ENV, "TRAVIS_PULL_REQUEST") && ENV["TRAVIS_PULL_REQUEST"] != "false"
```
Comment (oxinabox) [inline docs/make.jl]:
```suggestion
if get(ENV, "TRAVIS_PULL_REQUEST", "false") != "false"
```

**Comment by fredrikekre** [comment]
Documentation built successfully, a preview can be found here: https://fredrikekre.github.io/Literate.jl/preview-PR75

**Comment by fredrikekre** [comment]
$(body)

**Comment by fredrikekre** [comment]
Documentation built successfully, a preview can be found here: https://fredrikekre.github.io/Literate.jl/preview-PR101

**Comment by fredrikekre** [comment]
Implemented in Documenter now.

**Pull request closed without merging**